src/Umbraco.Web/Editors/AuthenticationController.cs

```diff
@@ -516,7 +516,8 @@ private string ConstructCallbackUrl(int userId, string code)
                 {
                     area = GlobalSettings.GetUmbracoMvcArea(),
                     u = userId,
-                    r = code
+                    r = code,
```

Comment:
We don't want to pass the security stamp separately, we don't want people to know about a security stamp at all. The `code` here should be the hash of the `code` + the current security stamp.

Reply:
Thanks will revert these changes back and look into the SecurityStamp stuff with the email from your notes 💪